tests/30rooms/21receipts.pl

```diff
@@ -48,7 +48,7 @@ sub find_receipt
 
          matrix_advance_room_receipt( $user, $room_id, "m.read" => $member_event_id )
       })->then( sub {
-         matrix_initialsync( $user )
+         matrix_initialsync( $user, limit => 1)
```

Comment:
See comment in earlier file for a neater idea.

Or keep this but fix the whitespace - `limit => 1 )`